scripts/display-travis-status.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const debug = require('debug')('display_travis_status')
+const pollTravis = require('../lib/pollTravis')
+
+module.exports = function (app) {
+  app.on('pull_request.opened', (event) => {
+    const owner = event.repository.owner.login
+    const repo = event.repository.name
+    if (repo !== 'nodejs.org') return
```

Comment:
We want to run this on multiple repos. citgm was enabled yesterday, and readable-stream will also get enabled as soon as #12 gets merged.